src/models/item.js

```diff
@@ -0,0 +1,54 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const itemSchema = new Schema({
+  name: {
+    type: String,
+    maxLength: [30, "item name's max length is 30 characters"],
+    required: [true, 'Item name is required'],
+  },
+  type: {
+    type: String,
+    enum: {
+      values: [
+        'Books',
+        'Stationery',
+        'School Books',
+        'Novels',
+        'Test Books',
+        'Furniture',
+      ],
+      message:
+        "Item type should be equal to 'Books', 'Stationery', 'School Books', 'Novels', 'Test Books' or 'Furniture'",
+    },
+    required: [true, 'Item type is required'],
+  },
+  description: {
+    type: String,
+    required: [true, 'Description is required'],
+  },
+  photo: {
+    type: String,
+    required: [true, 'Item photo is required'],
+  },
```

Comment:
You can add a custom validation here to ensure the value entered for photo is a real and valid URL. Otherwise people could enter any random string and that may affect the frontend developers in rendering the photo correctly.